library/pk.c

```diff
@@ -147,6 +147,40 @@ int mbedtls_pk_setup( mbedtls_pk_context *ctx, const mbedtls_pk_info_t *info )
 }
 
 #if defined(MBEDTLS_USE_PSA_CRYPTO)
+int mbedtls_pk_psa_err_translate( psa_status_t status )
```

Comment:
Preexisting, but since the function is being renamed anyway:

* Please don't abbreviate `error` to `err`. It only saves two characters but it's yet another thing to remember.
* The name doesn't convey in which direction the translation goes. Contrast with `mbedtls_to_psa_error`. I don't propose to just take the mirror image of that name because I see value in consistently using the `mbedtls_pk_` prefix. How about `mbedtls_pk_error_from_psa`?

Likewise for the ECP and RSA functions.

Reply:
@gilles-peskine-arm thanks, for RSA or ECP should it be `mbedtls_rsa_pk_error_from_psa` or simply `mbedtls_rsa_error_from_psa` ?
Since it can return PK errors in fallback, I don't know which one to use

Reply:
> Since it can return PK errors in fallback

You know what, I think this means the RSA/ECP functions should be `mbedtls_pk_xxx` and declared/defined in `pk*`. Thus: `mbedtls_pk_rsa_error_from_psa` (or `mbedtls_pk_error_from_psa_rsa`?).

Reply:
> declared/defined in pk*.

Yep, I'll move them back in `library/pk_wrap.h` & `library/pk_wrap.h`

> Thus: mbedtls_pk_rsa_error_from_psa (or mbedtls_pk_error_from_psa_rsa?).

Hard to choose, we translate PSA RSA errors to MbedTLS PK or RSA errors... the cleanest would be `mbedtls_pk_error_from_psa_rsa` then